### PR TITLE
fix(React): Fix Browse Pagination Bug

### PR DESCRIPTION
**Pull request by jjoyce0510**
**Scope**
This change fixes a bug in browse pagination, thanks to Joe!

**Changes**
Properly set total results & page for pagination so that the pagination works as expected. 

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: datahub-web-react/src/conf/Browse.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
 /*
     Number of results shown per browse results page
 */
-export const RESULTS_PER_PAGE = 20;
+export const RESULTS_PER_PAGE = 2;
```
Comment (shirshanka) [inline datahub-web-react/src/conf/Browse.tsx:4]: why 2?

File: datahub-web-react/src/app/search/EntityGroupSearchResults.tsx [hunk 2]
```diff
@@ -23,6 +22,8 @@ interface Props {
     query: string;
 }
 
+const RESULTS_PER_GROUP = 3;
```
Comment (shirshanka) [inline datahub-web-react/src/app/search/EntityGroupSearchResults.tsx:25]: why 3?

Comment (jjoyce0510) [reply datahub-web-react/src/app/search/EntityGroupSearchResults.tsx:25]: This means 3 entities per preview group (before you see "See all X results)... It looks good to me

File: datahub-web-react/src/conf/Browse.tsx (hunk 1 above)
Comment (jjoyce0510) [reply datahub-web-react/src/conf/Browse.tsx]: This is incorrect. Should match search at 10.

**Review by shirshanka — APPROVED** [review]
LGTM!

**Pull request merged**